### PR TITLE
normalize accepts floats and allows broadcast over channel dimension

### PR DESCRIPTION
**Pull request by dvd42**
Solves #230

### CODE REVIEW

File: kornia/color/normalize.py [hunk 1]
```diff
@@ -27,39 +28,50 @@ def forward(self, input: torch.Tensor) -> torch.Tensor:  # type: ignore
...
-        std (torch.Tensor): Standard deviations for each channel.
+        mean (torch.Tensor or float): Mean for each channel.
+        std (torch.Tensor or float): Standard deviations for each channel.
 
         Returns:
             torch.Tensor: The normalised image tensor.
     """
 
+    if isinstance(mean, float):
+        mean = torch.tensor([mean])  # prevent 0 sized tensors
```
Comment (edgarriba) [inline kornia/color/normalize.py]: name it `mean_tmp` same with std
